src/react-components/CHANGELOG.md

```diff
@@ -4,6 +4,27 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 Currently, this repo is in Prerelease.  When it is released, this project will adhere to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
+## [0.1.0]
+### Breaking Changes
```

Comment:
It's great that you are deprecating multiple components in favor of one component, but if it's a breaking change I think it's safe to completely remove those files. Also, this version bump isn't in `package.json`.